de.dlr.sc.virsat.concept.unittest.util/src/de/dlr/sc/virsat/concept/unittest/util/test/AConceptProjectTestCase.java

```diff
@@ -21,14 +24,40 @@
  */
 public abstract class AConceptProjectTestCase extends AProjectTestCase {
 
+	private static final String CONCEPT_ID_CORE = "de.dlr.sc.virsat.model.ext.core";
+	
 	/**
 	 * Method to load the test concept
 	 * @param pluginName The name of the plugin from which to load the concept 
 	 * @return the test concept
 	 */
 	protected Concept loadConceptFromPlugin(String pluginName) {
-	    String conceptXmiPluginPath = pluginName + "/concept/concept.xmi";
-		Concept concept = ConceptXmiLoader.loadConceptFromPlugin(conceptXmiPluginPath);
+		Concept concept = ActiveConceptConfigurationElement.loadConceptViaConceptName(pluginName);
 		return concept;
 	}
+	
+	/**
+	 * Adds the language core concept to the repository.
+	 * 
+	 * Requires addEditingDomainAndRepository() from super class to be executed before
+	 */
+	protected void activateCoreConcept() {
+		editingDomain.getCommandStack().execute(new RecordingCommand(editingDomain) {
+			@Override
+			protected void doExecute() {
+				addCoreConceptToRepository(repository);
+			}
+		});
+	}
+	
+	/**
+	 * Adds the language core concept to the repository (without using a command)
+	 * 
+	 * @param repository the repository in which the core concept should be added
+	 */
+	protected void addCoreConceptToRepository(Repository repository) {

```

Comment:
Why only core, why not a general method that can be used to add any concept to a test case?

Reply:
Probably that would also make sense but this method is needed in every test which uses concepts now...

Reply:
Apparantly @PhilMFischer got inspired by this and made a general method.
We should double check if the new method `loadConceptAndInstallToRepository ` already covers the loading use case here.

Reply:
I add an optional second parameter to also use it for other repositories... Now it does.